connector/connect/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -54,9 +55,9 @@ package object dsl {
       }
 
       def join(
-          otherPlan: proto.Relation,
-          joinType: JoinType = JoinType.JOIN_TYPE_INNER,
-          condition: Option[proto.Expression] = None): proto.Relation = {
+        otherPlan: proto.Relation,
```

Comment:
nit: 4 spaces indentation for parameter list.

Reply:
thanks.

My IDE always breaks it.... let me figure out a way to configure IDE to avoid this happening again.